tern/formats/default/generator.py

```diff
@@ -113,8 +114,16 @@ def get_layer_info_list(layer):
         file_level_licenses = ", ".join(layer_file_licenses_list)
 
     for package in layer.packages:
+        pkg_licenses = [package.pkg_license]
+        if package.pkg_format == 'deb':
+            # If many pkg_licenses, wrap the line for the table at 45 chars
+            pkg_licenses = wrap(", ".join(package.pkg_licenses), width=45) or ['']
```

Comment:
small nit: this line is longer than 72 characters. I suggest breaking it up just before `width=45`